reference/authorisation_model.md

```diff
@@ -0,0 +1,132 @@
+# JAAS: Authorisation Model
+
+JAAS uses a Relationship-Based Access Control (ReBAC) scheme for authorisation purposes. This document illustrates the underlying authorisation model used by JAAS.
+
+```{tip}
+For an explanation on Relationship-Based Access Control (ReBAC) check out :doc:`this <../explanation/authorisation>` explanation topic.
+```
+
+```{tip}
+As a reference on manipulating authorisation data, check out :doc:`this <./authorisation_data>` reference topic.
+```
+
+## The model
+
+JAAS authorisation model reshapes the `Juju permission model <https://juju.is/docs/juju/user-permissions>`_ into a ReBAC paradigm. The OpenFGA authorisation model used by JAAS is defined as:
```

Comment:
This link needs to be fixed. I suggest searching for all `_` and `__` to find rst links that weren't converted properly.

Reply:
grepped in each directory for this and fixed all that I could find.